RecoTauTag/RecoTau/test/runDeepTauIDsOnMiniAOD.py

```diff
@@ -12,6 +12,14 @@
 nThreads = 1
 
 process = cms.Process('TauID')
+## removing the following comments if running with SONIC
+## and comment out the previous line
+#from Configuration.ProcessModifiers.deepTauSonicTriton_cff import deepTauSonicTriton
+#process = cms.Process('TauID', deepTauSonicTriton)
+#process.load("HeterogeneousCore.SonicTriton.TritonService_cff")
+#process.TritonService.verbose = True
+#process.TritonService.fallback.enable = True
+
```

Comment:
Propose to add some logic instead of asking user to comment (out) part of the cfg. It can be done like this:
```
runWithSONIC = False # set true to run deepTauID with SONIC

process = None
if not runWithSONIC:
     process = cms.Process('TauID')
else:
     from Configuration.ProcessModifiers.deepTauSonicTriton_cff import deepTauSonicTriton
     process = cms.Process('TauID', deepTauSonicTriton)
     process.load("HeterogeneousCore.SonicTriton.TritonService_cff")
     process.TritonService.verbose = True
     process.TritonService.fallback.enable = True
```